src/dangerously-mutate.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Let's keep this in the same file as black-magic.

Reply:
Hm, for some reason I thought there was a ciclic dependency...